docs/getting_started/azure.md

```diff
@@ -0,0 +1,214 @@
+name must be unique among across multiple different Azure accounts.
+
+```bash
+$ az storage account create --name kopstest --resource-group kops-test
+```
+
+Set the env var `AZURE_STORAGE_ACCOUNT` to the storage account name for later use.
+
+```bash
+$ export AZURE_STORAGE_ACCOUNT=kopstest
+```
+
+Get an access key of the account and set it in env var `AZURE_STORAGE_KEY` for later use.
+
+```bash
+$ az storage account keys list --account-name kopstest
+[
+  {
+    "keyName": "key1",
+    "permissions": "Full",
+    "value": "RHWWn..."
+  },
+  {
+    "keyName": "key2",
+    "permissions": "Full",
+    "value": "..."
+  }
+
+]
+
+$ export AZURE_STORAGE_KEY="RHWWn...“
+```
+
+
+Then create a container.
+
+```bash
+$ az storage container create --name cluster-configs
+{
+  "created": true
+}
+```
+
+You can confirm that the container has been successfully created from
+Storage Exporter or by typing “az storage container list”.
+
+```bash
+$ az storage container list --output table
+Name             Lease Status    Last Modified
+---------------  --------------  -------------------------
+cluster-configs  unlocked        2020-10-06T21:12:36+00:00
+```
+
+## Step 3. Set up Credentials for kOps
+
+Type the following commands to generate credentials used for kOps.
+
+First, create a service principal in Active Directory.
+
+```bash
+$ az ad sp create-for-rbac --name kops-test --role owner --sdk-auth
+
+{
+  "clientId": "8c6fddb5...",
+  "clientSecret": "dUFzX1...",
+  "subscriptionId": "7e232...",
+  "tenantId": "76253...",
+  ...
+}
+```
+
+Set corresponding env vars:
+
+- Set `AZURE_TENANT_ID` to the `tenantId` of the output
+- Set `AZURE_CLIENT_ID` to the `clienteId` of the output
+- Set `AZURE_CLIENT_SECRET` to the `clientSecret` of the output.
+
+```bash
+$ export AZURE_TENANT_ID="76253..."
+$ export AZURE_CLIENT_ID="8c6fddb5..."
+$ export AZURE_CLIENT_SECRET="dUFzX1..."
+```
+
+## Step 4. Run kOps Commands
+
+Type the following command to create cluster configs and push them to
+Azure blob storage. `--state` specifies the container where cluster
+configs are stored. The command line flags starting from `--azure-`
+are for Azure specific configurations.
+
+```bash
+$ export KOPS_FEATURE_FLAGS=AlphaAllowAzure
+
+$ kops create cluster \
+  --cloud azure \
+  --name my-azure.k8s.local \
+  --state azureblob://cluster-configs \
+  --zones eastus-1 \
+  --network-cidr 172.16.0.0/16 \
+  --networking calico \
+  --azure-subscription-id "${AZURE_SUBSCRIPTION_ID}" \
+  --azure-tenant-id "${AZURE_TENANT_ID}" \
+  --azure-resource-group-name kops-test \
+  --azure-route-table-name kops-test \
+  --azure-admin-user ubuntu
+```
+
+We can confirm that config files are created in Blob storage.
+
+```bash
+$ az storage blob list --container-name cluster-configs --output table
+```
+
+Type the following command to build a k8s cluster.
+
+```
+$ kops update cluster  \
+  --name my-azure.k8s.local \
+  --state azureblob://cluster-configs \
+  --yes
+```
```

Comment:
More triple backtick fun

```suggestion
\```
$ kops update cluster  \
  --name my-azure.k8s.local
\```

Now add the `--yes` flag to have kOps provision the resources
and create the cluster. This will also add a kubeconfig context
for the cluster.

\```
$ kops update cluster  \
  --name my-azure.k8s.local \
  --yes
\```

```

I don't recall, does `kops validate cluster` work? If so we should instruct users to use that, otherwise we should add a note saying "it may take a few minutes for the cluster's API server to become reachable" Finally we should have them `kubectl get namespaces` or something to confirm they can reach the API server.

Reply:
Sure, `kops validate cluster` is not working. Let me add the description.